### DISCUSSION TITLE
platform/actors/development/actor-definition/dataset-schema

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#1185] [Comments]
# platform/actors/development/actor-definition/dataset-schema

Learn how to define and present your dataset schema in an user-friendly output UI.

https://docs.apify.com/platform/actors/development/actor-definition/dataset-schema

**Comment by AmmarSalmi** [comment]
I think this is a terrible way to go about explaining the label property. Devs will be confused and think the label is somehow related to the field type. You could use more generic labels like "Profile Pic" "Address" "Is Bookmarked" or any other labels to explain that they are just the headers of the fields in the output table.

```
"display": {
                        "component": "table",
                        "properties": {
                            "pictureUrl": {
                                "label": "Image",
                                "format": "image"
                            },
                            "linkUrl": {
                                "label": "Link",
                                "format": "link"
                            },
                            "textField": {
                                "label": "Text",
                                "format": "text"
                            },
                            "booleanField": {
                                "label": "Boolean",
                                "format": "boolean"
                            },
                            "arrayField": {
                                "label": "Array",
                                "format": "array"
                            },
                            "objectField": {
                                "label": "Object",
                                "format": "object"
                            },
                            "dateField": {
                                "label": "Date",
                                "format": "date"
                            },
                            "numericField": {
                                "label": "Number",
                                "format": "number"
                            }
```

**Comment by surajswaraj88-dotcom** [comment]
{
    "actorOutputSchemaVersion": 1,

    "title": "Cypress test report output",
    "description": "Test report from Cypress",
    "type": "object",

    "properties": {
        "reportUrl": {
            "type": "string",
            "title": "HTML Report",
            "template": "{{links.apiDefaultKeyValueStoreUrl}}/records/report.html"
        }
    }
}

**Comment by surajswaraj88-dotcom** [comment]
{
    "actorOutputSchemaVersion": 1,

    "title": "Cypress test report output",
    "description": "Test report from Cypress",
    "type": "object",

    "properties": {
        "reportUrl": {
            "type": "string",
            "title": "HTML Report",
            "template": "{{links.apiDefaultKeyValueStoreUrl}}/records/report.html"
        }
    }
}
